### PR TITLE
Pin ffi to 1.12.2 due to Windows crash

### PR DESCRIPTION
**Pull request by james-stocks**
See https://github.com/ffi/ffi/issues/784

Signed-off-by: James Stocks <jstocks@chef.io>

### CODE REVIEW

**Comment by james-stocks** [comment]
See the verify pipeline build history for the failures this addresses:
https://buildkite.com/chef-oss/inspec-inspec-master-verify/builds?branch=master

The bug manifests as InSpec quietly crashing on Windows for any command except `version`.

File: Gemfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ gem "inspec", path: "."
 # in it in order to package the executable. Hence the odd backwards dependency.
 gem "inspec-bin", path: "./inspec-bin"
 
-gem "ffi", ">= 1.9.14"
+gem "ffi", "1.12.2" # 1.13 does not work on Windows: https://github.com/ffi/ffi/issues/784
```
Comment (clintoncwolfe) [inline Gemfile]: Can we loosen this to [">= 1.9.14", "< 1.13"] to allow flexibility in resolving this when packaging?

Comment (james-stocks) [reply Gemfile]: Good call, updated.

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

**Comment by Schwad** [comment]
Thanks very much for this @james-stocks ! I just ran this code on a windows machine that was broken yesterday and now up and running 100%! 🎉  🙏

**Pull request merged**